x/bor/depinject.go

```diff
@@ -0,0 +1,5 @@
+package bor
+
+// IsOnePerModuleType implements the depinject.OnePerModuleType interface.
+func (am AppModule) IsOnePerModuleType() {
```

Comment:
Do we really need this? From what I remember, we decided not to use `depinject`, right?

Reply:
Yeah we are not atm, for sure
But we do need this method otherwise the module would not implement `appmodule.AppModule` as this method would be missing. Just placed it in the same place as other modules for consistency